posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "Appsilon and Sanofi joins the pharmaverse council!"
+author:
+  - name: Ari Siggaard Knoph
+description: "Updates to the pharmaverse council"
+date: "2024-04-29"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [community]
+# feel free to change the image
+image: "pharmaverse.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-04-29_appsilon_and__sa..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+We are excited to announce that **Appsilon** and **Sanofi** will take up the remaining two council seats on the pharmaverse council!
+
+Appsilon will on the council be represented by **[Damian Rodziewicz](https://www.linkedin.com/in/drodziewicz/)** and Sanofi by **[Andre Couturier](https://www.linkedin.com/in/andre-couturier-1800b447/)**.
+
+# Appsilon and Open Source
```

Comment:
could be worth adding each company's logo just for some imagery to mix up the blog